src/pages/blog/status-check-monitoring-with-playwright-on-checkly.mdx

```diff
@@ -0,0 +1,66 @@
+
+- Buy an item in your shop and validate that the user flow is working.
+- Validate that the OAuth login with e.g. Google is working.
+- Using a search on your website to ensure your elastic search in the background is correctly configured and serves the expected data.
+
+Checkly itself gives you the ability to run any Node.js related code either with Playwright or Puppeteer to automate your user flows. They have a secret store built in to provide the secrets over to your code once it is running.
+
+```js
+const assert = require("chai").assert;
+const playwright = require("playwright");
+
+const browser = await playwright.chromium.launch();
+const page = await browser.newPage();
+await page.goto("https://playwright.tech");
+
+assert.equal(await page.title(), "Everything about the Playwright framework - Playwright Community 🎭");
+await page.waitForSelector("text=Playwright Community");
+await page.waitForSelector("text=Try Playwright");
+assert.isAbove(await page.$$eval("article", elements => elements.length), 8)
+await browser.close();
+```
+
+In this example which we're ensuring the following checks:
+
+- Assert that the title has the expected value
+- Assert that "Playwright Community" is on the page
+- Assert that "Try Playwright" is on the page
+- Assert that at least 8 articles are on the page
+
+If one of them is failing, Checkly will automatically treat this as a failing test and alert the user by e.g. Slack, PagerDuty, SMS, or Email.
+
+## Integrations
+
+This at all replaces already quite a lot of common monitoring solutions. Instead of configuring complex user interfaces, you can create your own Node.js scripts that perform your status checks which will run in the cloud.
+
+For more advanced integration, Checkly provides support for GitHub, which gives you a lot of other integrations like with Vercel or Heroku. This could e.g. be used to run your end-to-end tests on a Pull Request level with an environment that was created by one of these PaaS (Platform as a Service) providers.
+
+<img src="/blog/status-check-monitoring-with-playwright-on-checkly/github-pr.png" style={{maxWidth: "min(100%, 500px)"}}/>
+
+## Summary
+
+Checkly provides an easy way of adding either status checks or entire end-to-end tests to your web applications which have now also Playwright support available. We at the Playwright Community are using it to ensure that the services which we publicly expose are on one hand high-available and on the other by that fully-functional when e.g. upgrading dependencies.
+
+We're looking forward to how Checkly will make their monitoring solution even more accessible for developers with e.g. versioned code, an integrated Monaco editor with better auto-completion, support for custom NPM modules or a better debugging experience. We would recommend giving it a try and have not to worry about where to run your status checks or end-to-end tests and benefit from their simplicity.
```

Comment:
If you think folks might be interesting in seeing how we are progressing on the items you have mentioned, you could point them to our public github roadmap: https://github.com/checkly/public-roadmap